cmd/forwarder/httpbin/httpbin.go

```diff
@@ -23,6 +24,10 @@ func (c *command) RunE(cmd *cobra.Command, args []string) error {
 	}
 	logger := stdlog.New(c.logConfig)
 
+	cmd.Flags().VisitAll(func(f *pflag.Flag) {
+		logger.Infof("%s %s", f.Name, f.Value)
```

Comment:
How about we did type assertions here to host any custom printing logic?

Reply:
It is difficult to break through all the `Value` interfaces and later grab a private field from `anyflag.Value`, but I did it. 
Please have a look.